### DISCUSSION TITLE
RyzenAI for Windows

### DISCUSSION THREAD

**Discussion by FlattusBlastus** [nextcloud/all-in-one#3878] [Wiki]
For those with RyzenAI capable processors, the 1.0 IPU driver was released 12/4/2023.
Get it here: https://ryzenai.docs.amd.com/en/latest/inst.html

RyzenAI (IPU driver) will never be available on Linux:
https://www.tomshardware.com/news/amd-asks-do-you-need-ryzen-ai-support-in-linux

You can set up AI services natively in Windows and then point the Nextcloud AI plugin at them so that AI queries run much faster.

**Comment by siccovansas** [comment]
FYI: Ryzen AI is available for Linux: https://www.phoronix.com/news/AMD-XDNA-Ryzen-AI-Driver-Patch and https://github.com/amd/xdna-driver

Does this mean that Nextcloud AI can make use of Ryzen AI on Linux already or are other steps necessary?

**Comment by FlattusBlastus** [comment]
The IPU driver is now the NPU driver. As noted by @siccovansas, RyzenAI is now available on Linux. I like the LocalAI AIO containers now.

**Reply by siccovansas** [reply]
Do you have RyzenAI working on Linux with Nextcloud using LocalAI??
